src/acl/external/LDAP_group/required.m4

```diff
@@ -5,4 +5,4 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-AS_IF([test "x$LIBLDAP_LIBS" != "x"],[BUILD_HELPER="LDAP_group"])
+AS_IF([test "x$LIBLDAP_LIBS" != "x"  -a "x$squid_host_os" != "xmingw"],[BUILD_HELPER="LDAP_group"])
```

Comment:
Out of scope change.

Reply:
This still needs to be removed from this PR.
FWIW, I am getting the same error in a number of other helpers that use LDAP. So I am a bit confused why you only have this one.